client-app/pages/account/list-details.vue

```diff
@@ -82,7 +82,7 @@
         </template>
 
         <!-- Empty list -->
-        <VcEmptyView v-else :text="$t('shared.wishlists.list_details.empty_list')">
+        <VcEmptyView v-else :text="$t('shared.wishlists.list_details.empty_list')" class="lg:mt-32">
```

Comment:
Why use such a large indent?  If you want to align an element in the middle of the screen, then it's better to do it more explicitly.

Reply:
there's no middle of the screen